### DISCUSSION TITLE
Multi-site installation with translations

### DISCUSSION THREAD

**Discussion by marcdap** [statamic/cms#6700] [Q&A] [answered]
Hello everyone,

We have a Statamic installation for managing different brands, for the same client, and it has the same fieldsets and a very similar page structure across all of them. 
Now we have a new requirement, one of these brands should be localizable. As we are using the Sites functionality to have different brands, we cannot add translations to one of them.

This is the current site's structure:

- Brand 1
- Brand 2
- Brand 3
  - en
  - de
  - es
  
And the page structure for each brand could be this one:

- Home
- Products
  - Product detail
- Blog
  - Post

Is there a way to add translations to sites? Do you have any recommendations?

Thank you very much.

**Comment by edalzell** [comment, marked as answer]
the above would be 6 sites:

1. brand-1
2. brand-2
3. brand-3-en
4. brand-3-de
5. brand-3-es

Then you can localize on the site handle as per the instructions.

**Reply by marcdap** [reply]
Thanks for your answer! I think it's the best approach. Maybe we should look at how we can filter brands with languages to avoid having a massive list of sites in case they implement more and more brands in the future.
